nilearn/plotting/glass_brain.py

```diff
@@ -145,7 +145,7 @@ def _get_object_bounds(json_content, transform):
     return xmin - xmargin, xmax + xmargin, ymin - ymargin, ymax + ymargin
 
 
-def plot_brain_schematics(ax, direction, **kwargs):
+def plot_brain_schematics(ax, direction,radiological=False, **kwargs):
```

Comment:
Still relevant. Also, we'd need to document `radiological` in the docstring if we add it to the parameters.

Reply:
There was a bug here with plot_brain_schematics, where omitting this radiological parameter automatically put it inside kwargs. kwargs is then passed onto a matplotlib function that didn't recognise radiological as a parameter and threw an error.

Is there a better solution? I'm still a bit unclear on how plot_glass_brain works because it's hard to see how the plot_glass_brain calls functions from glass_brain.py